src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -2863,6 +2863,11 @@ NeedsDistributedPlanning(Query *queryTree)
 
 	if (hasLocalRelation && hasDistributedRelation)
 	{
+		if (InsertSelectIntoLocalTable(queryTree))
+		{
+			ereport(ERROR, (errmsg("cannot plan INSERT INTO local_table SELECT ... FROM"
```

Comment:
I think this should be reworded as _cannot `INSERT` rows from a distributed query into a local table_. The `dist_table` and `local_table` identifiers are ad hoc and will be misleading appearing in SQL like that.